_data/reference.yaml

```diff
@@ -136,6 +136,8 @@ toc:
       path: /reference/pkg/nodejs/@pulumi/openstack/index.html
     - title: vsphere
       path: /reference/pkg/nodejs/@pulumi/vsphere/index.html
+    - title: Pulumi Webhooks
```

Comment:
Why is this underneath API Reference? 🤔 

It should probably be under Teams & Collaboration.